plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java

```diff
@@ -174,10 +174,16 @@
      * The OpenShift deploy timeout in seconds:
      * See this issue for background of why for end users on slow wifi on their laptops
      * DeploymentConfigs usually barf: https://github.com/openshift/origin/issues/10531
+     *
+     * Lets default to 2 hours which should be plenty of time on a slow connection to download
+     * all the images for a YAML/JSON manifest (which can contain many separate images)
      */
-    @Parameter(property = "fabric8.openshift.deployTimeoutSeconds")
+    @Parameter(property = "fabric8.openshift.deployTimeoutSeconds", defaultValue = "7200")
```

Comment:
I guess CI builds could easily override this timeout setting via `~/.m2/settings.xml` too - particularly as the CI job knows its already built the image so should have very fast download times? 

Unfortunately we use composition across maven modules to take individual templates and compose them into larger apps. I guess we could post process and keep increasing the timeouts for each app whenever it gets composed; but was hoping we could just pick a less-likely-to-fail default. After all folks could run 5 templates in parallel too to replicate the failure.